### ISSUE TITLE
org.id-lifdeposit: assign owner to community multi-sig

### ISSUE THREAD

**Issue by mfw78** [windingtree/org.id-lif-deposit#4]
**tl;dr**
Superuser functions such as `setWithdrawDelay()` are assigned to an EOA (nominally the old Winding Tree Deployer). This represents immediate medium level security risks.

**Overview**

The `LifDeposit` implementation contains two types of superuser:

1. `ProxyAdmin` - this is responsible for the token logic of the contract, and represents essentially root level access to the contract state by virtue of token logic modification. This is guarded by the WT Community multi-sig `0x876969b13dcf884C13D4b4f003B69229E6b7966A`.

2. `owner` - this is an account stored in state for the contract that allows `setWithdrawDelay()` on the contract - used for changing the delay period for locking up collateral. **This is currently set to the Winding Tree Deployer `0xead12c6ee973e03d2d630d81efa57f393cb9117f`.**

Hypothetically should the deployer EOA key be compromised, there would be no loss of funds, but there would be a denial of service, attack vector, allowing for either abnormally high withdraw delay (freezing funds).

Therefore it is **strongly recommended** to immediately execute the following:

```js
{
  chainId: 1,
  data: 0xf2fde38b000000000000000000000000876969b13dcf884c13d4b4f003b69229e6b7966a,
  from: 0xead12c6ee973e03d2d630d81efa57f393cb9117f,
  to: 0x7fa6ad0866caa4abde8d60b905b3ae60a3e0f014,
  value: { BigNumber: "0" },
}
```

**Comment by mfw78** [comment]
Ownership transferred to community multi-sig. Refer to https://etherscan.io/tx/0x10c1f5f2caa56eb5d7e5ba3bdb4d0d656141cbea2da17d532c1cc129bcba79ab.
